dvc/machine/backend/terraform.py

```diff
@@ -3,8 +3,9 @@
 from functools import partial, partialmethod
 from typing import TYPE_CHECKING, Iterator, Optional
 
+from dvc_ssh import DEFAULT_PORT, SSHFileSystem
```

Comment:
Looks like we don't even need dvc_ssh here, and could use sshfs directly. CC @pmrowla @karajan1001 

Keeping as is for now, but this also means that we have (and had) a hidden dependency on sshfs (and now on dvc_ssh).